tests/client/conftest.py

```diff
@@ -69,7 +69,7 @@ def is_using_real_services(request):
 
 @pytest.fixture(scope='session')
 def client():
-    return ScrapinghubClient(auth=TEST_ADMIN_AUTH,
+    return ScrapinghubClient(apikey=TEST_ADMIN_AUTH,
```

Comment:
Added, please review [the changes](https://github.com/scrapinghub/python-scrapinghub/pull/48/commits/d5ef2c3ec84d3b8c48109efeb8e1c83b802e4ece).

Now it works in the following manner (I added a warning when using user/pass pair in legacy `Connection`): with a given `SHUB_JOBAUTH` env var, a client can be instantiated with:
```
c = ScrapinghubClient(os.env['SHUB_JOBAUTH']))
UserWarning: A lot of endpoints support authentication only via apikey.
```
After it you can use the client as usual, but it have access only to job data for now.